Recrutify/Recrutify.DataAccess/Models/AppointInterview.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Recrutify.DataAccess.Models
+{
+    public class AppointInterview
```

Comment:
we don't name models as verbs. InterviewAppointment same for corresponding dtos and vars